server/routerlicious/packages/routerlicious/src/kafka-service/command.ts

```diff
@@ -23,8 +24,10 @@ export function execute(
         .arguments("<name> <lambda>")
         .action((name: string, lambda: string) => {
             configureLogging(configOrPath);
-
+            const lumberjackEngine = new WinstonLumberjackEngine();
+            Lumberjack.setup([lumberjackEngine]);
```

Comment:
Is there a schema that could make sense to all Kafa-services, like Deli, Scribe, Scriptorium, etc? For example, if all of them know about `tenantId` and `documentId`, we can define a schema for that and specify it here.